modules/tracker/models/base/TrendModelBase.php

```diff
@@ -82,9 +82,10 @@ public function __construct()
      * @param null|int $configItemId configuration item id
      * @param null|int $testDatasetId test dataset item id
      * @param null|int $truthDatasetId truth dataset item id
+     * @param null|string $unit scalar value unit
      * @return false|Tracker_TrendDao trend DAO or false if none exists
      */
-    abstract public function getMatch($producerId, $metricName, $configItemId, $testDatasetId, $truthDatasetId);
+    abstract public function getMatch($producerId, $metricName, $configItemId, $testDatasetId, $truthDatasetId, $unit);
```

Comment:
We should probably maintain the previous API, so give `$unit` a default value. I favor `false`. Do the same in any other public functions.